pkg/liqonet/natmappinginflater/natMappingInflater_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/dynamic/fake"
 
-	netv1alpha1 "github.com/liqotech/liqo/apis/net/v1alpha1"
+	liqonetapi "github.com/liqotech/liqo/apis/net/v1alpha1"
```

Comment:
Why did you change the import name?

Reply:
GCI tool reported to me there were 2 identical imports called with a different name. So I have removed one of these. I don't know why in the old version it only reports one now. Maybe I will try to change again the name. Thank you